### PR TITLE
add editorial for cf 1368D

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: solutions/silver/cf-1368D.mdx
Comment (ryanchou-dev) [inline solutions/silver/cf-1368D.mdx:17]: it'd be great if you could structure these as hints! (see https://github.com/cpinitiative/usaco-guide/pull/3430)

File: solutions/silver/cf-1368D.mdx
Comment (ryanchou-dev) [inline solutions/silver/cf-1368D.mdx:50]: make multiline + even lines out?

**Review by ryanchou-dev — COMMENTED** [review]
awesome :O

File: solutions/silver/cf-1368D.mdx
Comment (SansPapyrus683) [reply solutions/silver/cf-1368D.mdx:50]: ^

File: solutions/silver/cf-1368D.mdx
Comment (SansPapyrus683) [inline solutions/silver/cf-1368D.mdx:74]:
```suggestion
public class AndOrSquareSum {
```

File: solutions/silver/cf-1368D.mdx
Comment (SansPapyrus683) [inline solutions/silver/cf-1368D.mdx:76]: use bufferedreader/kattio for consistency

File: solutions/silver/cf-1368D.mdx
Comment (SansPapyrus683) [inline solutions/silver/cf-1368D.mdx:128]: put in constant

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
poggies

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:14]: > the number and position of the set bits don't change
and
> Before and after the operation, there are two $1$ bits that are set, one $2$ bit that is set, and one $4$ bit that is set.
doesn't really make sense

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:12]: It'd be great if you could prove this claim

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:20]: > To structure this array optimally, move all the set bits to one end.

What does set bits mean and also explain exactly how the array is reconfigured given you provided an example

Comment (jessechoe10) [reply solutions/silver/cf-1368D.mdx:20]:
```suggestion
The optimal configuration can be built by handling each bit separately. This approach takes into account Observation 2, which states that no operations will alter the bits. For example, consider the array, `[1, 5, 4, 7, 3]`. In binary form, it is represented as `[001, 101, 100, 111, 011]`. To structure this array optimally, move all the set bits to one end. This results in the reconfigured array: `[000, 001, 101, 111, 111]`. In decimal form, this new array reads `[0, 1, 5, 7, 7]`. Finally, find the sum of squares of the elements in this new array. The resulting total provides the desired solution.
```

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:41]: Reword this

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:91]: multiline + make each line even

**Comment by 876pol** [comment]
i made a few changes to the explanation - lmk if some parts are still unclear

File: solutions/silver/cf-1368D.mdx
Comment (ryanchou-dev) [inline solutions/silver/cf-1368D.mdx:184]: mutiline comment this

Comment (SansPapyrus683) [reply solutions/silver/cf-1368D.mdx:184]: if it's two lines it's fine

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:30]:
```suggestion
Notice that before and after each operation, the number of active bits at each position don't change.
```

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:36]: same here

File: solutions/silver/cf-1368D.mdx
Comment (jessechoe10) [inline solutions/silver/cf-1368D.mdx:38]: and here

File: solutions/silver/cf-1368D.mdx
Comment (876pol) [reply solutions/silver/cf-1368D.mdx]: i wasn't too sure how to draw it out as a venn diagram, but i added a truth table to make it more clear

**Comment by SansPapyrus683** [comment]
image

💀

**Comment by 876pol** [comment]
😳 should work now

**Review by jessechoe10 — APPROVED** [review]
I'll wait for Sans to approve before merging.

**Review by ryanchou-dev — COMMENTED** [review]
would be cool if you could use `&` and `|` to keep it consistent!

other than that, looks good. we rly don't need 3 content managers to approve, so ill merge afterwards :DD

**Pull request merged**
